### PR TITLE
[Sentry][Native] Update version 0.4.8 -> 0.4.9

### PR DESCRIPTION
**Pull request by themarwhal**
Signed-off-by: Marie Bremner <marwhal@fb.com>

## Summary
Update Sentry Native version from 0.4.8 -> 0.4.9

## Test Plan
sentry still builds
MME/SessionD still builds and runs with Sentry

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: lte/gateway/deploy/roles/dev_common/tasks/main.yml
Comment (pshelar) [inline lte/gateway/deploy/roles/dev_common/tasks/main.yml:276]: currently we have separate ways to install sentry in production and dev environment. It would be nice to use same mechanism for both. Can you add it ad TODO.

**Review by pshelar — APPROVED** [review]

File: lte/gateway/deploy/roles/dev_common/tasks/main.yml
Comment (themarwhal) [reply lte/gateway/deploy/roles/dev_common/tasks/main.yml:276]: how do we install sentry for prod? @pshelar 
Yeah will add a todo

Comment (pshelar) [reply lte/gateway/deploy/roles/dev_common/tasks/main.yml:276]: package is built using: https://github.com/magma/magma/blob/master/third_party/build/bin/sentry-native-build.sh
This is deployed in production. we could deploy same in vagrant VM.

**Comment by ekowtaylor** [comment]
Once this PR has been approved and passing all CI checks, use the ready2merge label to indicate the maintainers can merge your PR.

**Pull request merged**
